Script/AtomicEditor/ui/frames/menus/MainFrameMenu.ts

```diff
@@ -40,7 +40,8 @@ class MainFrameMenu extends Atomic.ScriptObject {
         MenuItemSources.createMenuItemSource("menu tools", toolsItems);
         MenuItemSources.createMenuItemSource("menu developer", developerItems);
         MenuItemSources.createMenuItemSource("menu help", helpItems);
-
+        this.goScreenshot = 0;
+        this.subscribeToEvent(Atomic.UpdateEvent((ev) => this.handleScreenshot(ev)));
```

Comment:
I think I'm caught up... Duktape has no native setTimeout function, like you see all over the lying internet. Luckily, svaarala, Jay & TSH had foreseen this deficiency, and provided native functions with the AtomicEventLoop module.